lib/auth.coffee

```diff
@@ -340,6 +340,7 @@ getAuth = (deps, opts) ->
 	# @param {Object} [credentials={}] - in the form of username, password and email
 	# @param {String} credentials.email - the email
 	# @param {String} credentials.password - the password
+	# @param {(String|undefined)} [credentials.'g-recaptcha-response'] - the captcha response
```

Comment:
I was originally trying to do `[billingInfo['g-recaptcha-response']]` or `[billingInfo["g-recaptcha-response"]]`, but jsdoc didn't like them and thought it was a separate argument. After some googling it seems that there is an ongoing discussion on this.

Reply:
Interesting, I didn't notice that TBH, I just assumed that is the syntax.